dao/src/main/kotlin/tables/runs/shared/IssuesTable.kt

```diff
@@ -45,7 +47,7 @@ class IssueDao(id: EntityID<Long>) : LongEntity(id) {
     companion object : SortableEntityClass<IssueDao>(IssuesTable) {
         fun findByIssue(issue: Issue): IssueDao? = find {
             IssuesTable.issueSource eq issue.source and
-                    (IssuesTable.message eq issue.message) and
+                    (DigestFunction(IssuesTable.message) eq DigestFunction(stringLiteral(issue.message))) and
```

Comment:
Same here, I would put the test with the function call as the last one.

Reply:
As above.